mayastor/src/grpc/server.rs

```diff
@@ -19,13 +24,16 @@ impl MayastorGrpcServer {
         rpc_addr: String,
     ) -> Result<(), ()> {
         info!("gRPC server configured at address {}", endpoint);
+        let address = rpc_addr.clone();
         let svc = Server::builder()
             .add_service(MayastorRpcServer::new(MayastorSvc::new(
                 Duration::from_millis(4),
             )))
             .add_service(BdevRpcServer::new(BdevSvc::new()))
-            .add_service(JsonRpcServer::new(JsonRpcSvc {
-                rpc_addr,
+            .add_service(BdevRpcServer_v1::new(BdevSvc_v1::new()))
```

Comment:
How about instead of import v1::BdevRpcServer as BdevRpcServer_v1 we simply change these call sites to:

```rust
.add_service(v1::BdevRpcServer::new(.....))
```

Reply:
We will have to use it like this if we dont import. since BdevRpcServer comes inside `bdev_rpc_server`
```
.add_service(rpc::mayastor::v1::bdev_rpc_server::BdevRpcServer::new(BdevSvc_v1::new()))
```

Reply:
@gila PTAL, if this is how we need it. 

IMHO, i find importing and using it as `BdevRpcServerV1` much easier to read.

Reply:
The original proto files data back from https://github.com/tower-rs/tower-grpc. We are really close... but a few things to consider remain to clean up.

Reply:
Didnt get the comment @gila . What should we clean up?

Reply:
As discussed, 
The long module name is actually coming from the autogenerated code 
```
rpc::mayastor::v1::bdev_rpc_server::BdevRpcServer
```

so how should/can we change that?